packages/ui/src/domainModules/collectionMammals/components/MammalManager/index.js

```diff
@@ -399,6 +414,9 @@ class MammalManager extends Component {
         if (items && items.length) {
           this.props.setCurrentTableRowNumber(1)
           this.props.setFocusedSpecimenId(items[0].id)
+        } else {
+          this.props.setCurrentTableRowNumber(undefined)
+          this.props.setFocusedSpecimenId(undefined)
```

Comment:
need to use `del` instead `set` action creators for those two. so you need to add them in `mapDispatchToProps` and use them here